.github/workflows/engine_coverage.yml

```diff
@@ -189,11 +189,11 @@ jobs:
             local coverage_filtered_file="${{github.workspace}}/${{env.BUILD_PATH}}/coverage_${module}_filtered.info"
 
             # Generate the coverage report
-            lcov  --no-external --capture --initial --directory ${module_build_path} --base-directory ${module_path} --exclude "${module_path}/test/*" --output-file ${coverage_base_file}
+            lcov  --no-external --capture --initial --directory ${module_build_path} --base-directory ${module_path} --exclude "${module_path}/test/*" --output-file ${coverage_base_file} --ignore-errors unused,mismatch
```

Comment:
I've added this flags to ignore some errors:
- unused: the coverage reports that the keystore test folder exclusion is unused
- mismatch: the GTest macros create a number line difference that causes confusion to `lcov`

Reply:
I've removed this.
Thank you @juliancnn